corehq/apps/hqwebapp/tables/export.py

```diff
@@ -152,13 +152,19 @@ def reconstruct_from_export_context(cls, context):
 
         request = HttpRequest()
         request.method = 'GET'
-        request.GET.update(context['request_params'])
+
+        query_dict = QueryDict(mutable=True)
+        for key, values in context['request_params'].items():
+            query_dict.setlist(key, values)
+        request.GET = query_dict
+
```

Comment:
`QueryDict.urlencode()` will serialize the value to a string that can be passed back to the QueryDict constructor.

```suggestion
        request.GET = QueryDict(context['request_params'])
```